testsuite/src/Test/Tasty/Clash.hs

```diff
@@ -754,7 +754,7 @@ netlistTest' env target extraGhcArgs modName funcName path =
              , "runghc"
              ]
              ++ langExts ++
-             [ "-DNETLISTTEST"
+             [ "-DCLASHLIBTEST"
```

Comment:
is this used somewhere?

Reply:
Output tests define a CPP symbol `OUTPUTTEST`, so when netlist tests were created they defined a similar symbol. I'm not actually sure if either are used, I don't think I've ever seen 

```c
#if OUTPUTTEST
...
#endif
```

in any files